cedar-policy-validator/src/validation_result.rs

```diff
@@ -125,22 +125,22 @@ impl<'a> SourceLocation<'a> {
 #[cfg_attr(test, derive(Eq, PartialEq))]
 #[non_exhaustive]
 pub enum ValidationErrorKind {
-    /// An entity type was seen in a policy but was not found in the schema.
+    /// A policy contains an entity type that is not declared in the schema.
     #[error(
-        "Unrecognized entity type {}{}",
+        "unrecognized entity type `{}`{}",
         .0.actual_entity_type,
         match &.0.suggested_entity_type {
-            Some(s) => format!(", did you mean {}?", s),
+            Some(s) => format!(", did you mean `{}`?", s),
             None => "".to_string()
         }
     )]
     UnrecognizedEntityType(UnrecognizedEntityType),
-    /// An action id was seen in a policy but was not found in the schema.
+    /// A policy contains an action id that is not declared in the schema.
```

Comment:
Is the phrase "action id" consistent with our other error messages? Searching for "entity id" doesn't find many results, so maybe just "action" or "action entity" would be better.

Reply:
Good catch. Fixed.